yahoo_fantasy_api/league.py

```diff
@@ -552,6 +552,30 @@ def percent_owned(self, player_ids):
             pass
         return po
 
+
+    def ownership(self, player_ids):
+        """Retrieve the owner of a player
+
+        :param player_ids: Yahoo! Player IDs to retrieve owned for
+        :type player_ids: list(int)
+        :return: Ownership status of player
+        :rtype: dict
+
+        >>> lg.player_owner(1, [3737])
+        {"3737" : {"ownership_tpye" : "team", "owner_team_name": "team name"}}
```

Comment:
I like that you have included sample code, but it seems like it is out of date.  Can you update it to reflect the current API name?